content/python/concepts/modules/modules.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: "Modules" 
+Subjects:
+  - "Computer Science"
+  - "Data Science"
+Tags: 
+  - "Libraries"
+  - "Modules"
+  - "Scope"
+CatalogContent: 
+  - "learn-python-3"
+  - "paths/computer-science"
+---
+
+A module is a file that contains Python functions, definitions, and statements that can be included in an application.
```

Comment:
I would move "Python" over in between "a" and "file" and then omit "Python file" below on line 19.

Reply:
Good idea. Done!